packages/account-v2/src/router/constants/routesConfig.tsx

```diff
@@ -0,0 +1,86 @@
+import DummyRoute from '../components/dummy-route/dummy-route';
+
+export const routes = [
```

Comment:
may I clarify, should we also add here subroutes?

Reply:
Cashier team is working on the page layout. Once thats done , we can set categories